.github/workflows/gpu.yml

```diff
@@ -1,6 +1,3 @@
-# This workflow builds and tests providers using GPU memory. It requires 
-# "level_zero" labeled self-hosted runners installed on systems with the 
-# appropriate GPU and drivers.
```

Comment:
Why did you remove that?

Reply:
restored + changed "level_zero labeled" part